files/build_templates/syncd.service.j2

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=syncd service
+Requires=database.service updategraph.service
+{% if sonic_asic_platform == 'broadcom' %}
+Requires=opennsl-modules-4.9.0-7-amd64.service
+{% elif sonic_asic_platform == 'nephos' %}
+Requires=nps-modules-4.9.0-7-amd64.service
+{% endif %}
+After=database.service updategraph.service
+After=interfaces-config.service
+{% if sonic_asic_platform == 'broadcom' %}
+After=opennsl-modules-4.9.0-7-amd64.service
+{% elif sonic_asic_platform == 'nephos' %}
+After=nps-modules-4.9.0-7-amd64.service
```

Comment:
This is a great question!

My tentative thought is that they both needs this 'After', Because they can be started independently now. Or at least it doesn't hurt for them both having it?

Reply:
I think swss does not have dependency on those?

Reply:
But swss has dependency on syncd, and syncd has dependency on above modules.

Since we are not really removing the dependency between swss and syncd, it makes more sense to keep the dependency.

Do you agree?

Reply:
Thanks Jipan and Taoyu, I've removed these lines.